conformance/fhir-ig-davinci-hrex/src/test/resources/JSON/020/Bundle-subscription-notification-with-data.json

```diff
@@ -51,7 +51,7 @@
icationRequest/57\">http://example.org/new-payer/CommunicationRequest/57</a></p><p><b>status</b>: completed</p><p><b>intent</b>: order</p><p><b>code</b>: data request task <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"CodeSystem-hrex-temp.html\">HRex Temporary Code System</a>#data-request)</span></p><p><b>for</b>: <a href=\"http://example.org/new-payer/fhir/Patient/pat1\">http://example.org/new-payer/fhir/Patient/pat1</a></p><p><b>authoredOn</b>: 2020-07-30 08:07:15-0600</p><p><b>requester</b>: <a href=\"http://example.org/new-payer/fhir/Organization/hl7pay\">http://example.org/new-payer/fhir/Organization/hl7pay</a></p><p><b>owner</b>: <a href=\"http://example.org/provider/fhir/Organization/hl7prov\">http://example.org/provider/fhir/Organization/hl7prov</a></p><p><b>reasonCode</b>: Support prior authorization decision-making <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> ()</span></p><p><b>reasonReference</b>: <a href=\"http://example.org/new-payer/fhir/Claim/123\">http://example.org/new-payer/fhir/Claim/123</a></p><h3>Inputs</h3><table class=\"grid\"><tr><td>-</td><td><b>Type</b></td><td><b>Value[x]</b></td></tr><tr><td>*</td><td>data query input <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"CodeSystem-hrex-temp.html\">HRex Temporary Code System</a>#data-query)</span></td><td>Condition?patient=Patient/pat1&amp;clinical-status=active&amp;code:below=http%3A%2F%2Fsnomed.info%2Fsct%7C73211009</td></tr></table><h3>Outputs</h3><table class=\"grid\"><tr><td>-</td><td><b>Type</b></td><td><b>Value[x]</b></td></tr><tr><td>*</td><td>data values output <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"CodeSystem-hrex-temp.html\">HRex Temporary Code System</a>#data-value)</span></td><td><a name=\"results\"> </a><blockquote><div style=\"display: inline-block; background-color: #d9e0e7; padding: 6px; margin: 4px; border: 1px solid #8da1b4; border-radius: 5px; line-height: 60%\"><p style=\"margin-bottom: 0px\">Resource \"results\" </p></div><p><b>type</b>: searchset</p><p><b>total</b>: 1</p><h3>Links</h3><table class=\"grid\"><tr><td>-</td><td><b>Relation</b></td><td><b>Url</b></td></tr><tr><td>*</td><td>self</td><td><a href=\"http://example.org/new-payer/fhir/Condition?patient=Patient/pat1&amp;clinical-status=active&amp;code:below=http%3A%2F%2Fsnomed.info%2Fsct%7C73211009\">http://example.org/new-payer/fhir/Condition?patient=Patient/pat1&amp;clinical-status=active&amp;code:below=http%3A%2F%2Fsnomed.info%2Fsct%7C73211009</a></td></tr></table><blockquote><p><b>entry</b></p><p><b>fullUrl</b>: <a href=\"http://local.fhir.org:960/r4/Condition/example\">http://local.fhir.org:960/r4/Condition/example</a></p><h3>Searches</h3><table class=\"grid\"><tr><td>-</td><td><b>Mode</b></td></tr><tr><td>*</td><td>match</td></tr></table></blockquote></blockquote></td></tr></table></div>"
```

Comment:
i'd like these paths updated from JSON/020 to JSON/100 as well

Reply:
done